tests/integration/workflows/test_workflows_tasks_upload.py

```diff
@@ -233,7 +232,9 @@ def test_store_record_inspirehep_api_literature_new(workflow_app):
                 headers={'content-type': 'application/json'},
                 status_code=201,
                 json={
-                    'control_number': expected_control_number,
+                    "metadata": {
```

Comment:
why now is different? :/ we haven't changed anything

Reply:
It's not different, previously it was wrong.

Reply:
how it was passing the tests :/

Reply:
Data I copied as a mocked response from inspirehep were wrong. It was my mistake. When I was copying it i missed one level of nested data.